.github/workflows/pytest.yml

```diff
@@ -56,7 +56,7 @@ jobs:
 
       - name: run pytest
         run: |
-          pytest -n 2 --runslow
+          pytest -n 2 --runslow --dist=loadfile
```

Comment:
should we put this as default in `setup.cfg` ?

Reply:
I just tested this distribution method with the notebooks and it works. But if we put this option, we would also have to specify a parallel execution method (e.g. the number of nodes --n auto or --n1 etc.). Is this still an option for setup.cfg then?

Reply:
There are some design patterns in test_transformations, which prohibit to run the tests in an arbitrary parallel order. So we have to enforce this particular distribution method.

Reply:
@vhirtham fyi

Reply:
My only concern about adding this as a default option is that we would also need to add an option for the number of hosts (e.g. -n1 etc.), because without it you receive an error message like this:
```ERROR: MISSING test execution (tx) nodes: please specify --tx```

Reply:
yes if this need `-n` than keep the current implementation specified in GitHub actions 👍

Reply:
I decided to put -n1 as default, which can be overridden from the outside. Do you agree?

Reply:
yes that is a nice solution!

Reply:
I noticed a problem with PyCharm integration
When using `xdist` the debugger does not attach to the test instances, anyone else? This happens even for using a single xdist runner :(

So in that case I would suggest to remove xdist arguments from the default config to make it easier to use (not ideal I know)

Reply:
Actually PyCharms debugger should work fine with subprocesses. If multiple test workers are an issue for the debugger, I agree to be explicit for this option in CI environments.

Reply:
I'm not sure what works exactly, we can take a look at it together tomorrow :)